embedded-pulsar/README.adoc

```diff
@@ -0,0 +1,34 @@
+=== embedded-pulsar
+
+==== Maven dependency
+
+.pom.xml
+[source,xml]
+----
+<dependency>
+    <groupId>com.playtika.testcontainers</groupId>
+    <artifactId>embedded-pulsar</artifactId>
+    <scope>test</scope>
+</dependency>
+----
+
+==== Consumes (via `bootstrap.properties`)
+
+* `embedded.pulsar.enabled` `(true|false, default is 'true')`
+* `embedded.pulsar.imageTag` `(default is '2.6.1')`
+** You can pick the desired version on https://hub.docker.com/r/apachepulsar/pulsar/tags[dockerhub]
+
+==== Produces
+
+* `embedded.pulsar.brokerUrl`
+* `embedded.pulsar.httpServiceUrl`
+
+==== Example
+
+Properties can be used to configure `PulsarClient` directly :
```

Comment:
this section seems to be missing

Reply:
I looked at other modules and example section contained spring properties configured with embedded ones. However for pulsar there is no Spring module so I did not know what to put here,